### PR TITLE
[13.0] [MIG] stock_picking_operation_quick_change: Migration to v13.0

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT27875

please @joao-p-marques @victoralmau review this

### CODE REVIEW

File: stock_picking_operation_quick_change/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# © 2017 Sergio Teruel <sergio.teruel@tecnativa.com>
```
Comment (victoralmau) [inline stock_picking_operation_quick_change/__manifest__.py]:
```suggestion
# © 2017 Tecnativa - Sergio Teruel
```

File: stock_picking_operation_quick_change/tests/test_stock_picking_operation_quick_change.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
+# © 2017 Sergio Teruel <sergio.teruel@tecnativa.com>
```
Comment (victoralmau) [inline stock_picking_operation_quick_change/tests/test_stock_picking_operation_quick_change.py]:
```suggestion
# © 2017 Tecnativa - Sergio Teruel
```

File: stock_picking_operation_quick_change/views/stock_picking_view.xml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2017 Sergio Teruel <sergio.teruel@tecnativa.com>
```
Comment (victoralmau) [inline stock_picking_operation_quick_change/views/stock_picking_view.xml]:
```suggestion
<!-- Copyright 2017 Tecnativa - Sergio Teruel
```

File: stock_picking_operation_quick_change/wizards/stock_picking_wizard.py [hunk 4]
```diff
@@ -0,0 +1,103 @@
+# © 2017 Sergio Teruel <sergio.teruel@tecnativa.com>
```
Comment (victoralmau) [inline stock_picking_operation_quick_change/wizards/stock_picking_wizard.py]:
```suggestion
# © 2017 Tecnativa - Sergio Teruel
```

File: stock_picking_operation_quick_change/wizards/stock_picking_wizard.py [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+class StockPickingOperationWizard(models.TransientModel):
+    _name = "stock.picking.operation.wizard"
+    _description = "Stock Picking Operation Wizard"
+
+    def _prepare_default_values(self, picking):
+        return {"location_dest_id": picking.location_dest_id.id}
+
+    @api.model
+    def default_get(self, fields):
+        res = super(StockPickingOperationWizard, self).default_get(fields)
```
Comment (victoralmau) [inline stock_picking_operation_quick_change/wizards/stock_picking_wizard.py]:
```suggestion
        res = super().default_get(fields)
```

File: stock_picking_operation_quick_change/wizards/stock_picking_wizard_view.xml [hunk 6]
```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2017 Sergio Teruel <sergio.teruel@tecnativa.com>
```
Comment (victoralmau) [inline stock_picking_operation_quick_change/wizards/stock_picking_wizard_view.xml]:
```suggestion
<!-- Copyright 2017 Tecnativa - Sergio Teruel
```

**Review by victoralmau — CHANGES_REQUESTED** [review]
Code review OK, only some little changes.

**Comment by CarlosRoca13** [comment]
@victoralmau changes done

**Review by victoralmau — APPROVED** [review]

File: stock_picking_operation_quick_change/tests/test_stock_picking_operation_quick_change.py [hunk 7]
```diff
@@ -93,15 +103,15 @@ def test_picking_operation_change_location_dest_all(self):
...
+        ).create({"new_location_dest_id": new_location_dest_id.id, "change_all": True})
         move_lines = self.picking.mapped("move_line_ids")
         self.assertEqual(wiz.location_dest_id, self.picking.location_dest_id)
         self.assertEqual(wiz.old_location_dest_id, move_lines[:1].location_dest_id)
         wiz.action_apply()
         move_lines = self.picking.mapped("move_line_ids.location_dest_id")
         self.assertEqual(len(move_lines), 1)
 
-    def test_picking_operation_change_location_dest(self):
+    def _test_picking_operation_change_location_dest(self):
```
Comment (joao-p-marques) [inline stock_picking_operation_quick_change/tests/test_stock_picking_operation_quick_change.py]: Are these tests meant to be skipped?

File: stock_picking_operation_quick_change/tests/test_stock_picking_operation_quick_change.py [hunk 8]
```diff
@@ -129,7 +139,7 @@ def test_picking_operation_change_location_dest(self):
         move_lines = self.picking.mapped("move_line_ids.location_dest_id")
         self.assertEqual(len(move_lines), 2)
 
-    def test_picking_operation_change_location_dest_failed(self):
+    def _test_picking_operation_change_location_dest_failed(self):
```
Comment (joao-p-marques) [inline stock_picking_operation_quick_change/tests/test_stock_picking_operation_quick_change.py]: Same here as above

**Review by joao-p-marques — COMMENTED** [review]
Code review. Waiting for runbot for functional review :sweat_smile:

**Comment by CarlosRoca13** [comment]
UPS sorry, changed @joao-p-marques

**Review by joao-p-marques — APPROVED** [review]

**Pull request merged**